app/views/finance/financial_transactions/new_collection.html.haml

```diff
@@ -16,16 +17,23 @@
       });
 
       $(document).on('touchclick', 'a[data-add-all-ordergroups]', function() {
-        var value = prompt("#{escape_javascript(heading_helper(FinancialTransaction, :amount))}:");
-        if (value === null)
-          return false;
+        var customField = $(this).data('custom-field');
```

Comment:
so `customField` is the name of the field to use.  

how does `custom-field` get set?  (in which code)